doc/modules/linear_model.rst

```diff
@@ -1581,12 +1647,14 @@ where
 .. math::
 
   H_{\epsilon}(z) = \begin{cases}
-         z^2, & \text {if } |z| < \epsilon, \\
-         2\epsilon|z| - \epsilon^2, & \text{otherwise}
+        z^2, & \text {if } |z| < \epsilon, \\
+        2\epsilon|z| - \epsilon^2, & \text{otherwise}
   \end{cases}
 
 It is advised to set the parameter ``epsilon`` to 1.35 to achieve 95% statistical efficiency.
 
+|details-end|
+
```

Comment:
Move the examples and reference below above here (above the Notes section).
We can make a dropdown menu "reference".
The examples should be moved just before the mathematical details.